.github/workflows/zeebe-aws-os-dispatch-tests.yml

```diff
@@ -0,0 +1,109 @@
+    # see https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#exit-codes-and-error-action-preference
+    shell: bash
+
+env:
+  DOCKER_PLATFORMS: "linux/amd64,linux/arm64"
+
+jobs:
+ integration-tests:
+    name: "[IT] ${{ matrix.name }}"
+    timeout-minutes: 15
+    runs-on: ${{ matrix.runs-on }}
+    strategy:
+      fail-fast: false
+      matrix:
+        group: [ os-exporter, camunda-exporter ]
+        include:
+          - group: os-exporter
+            name: "Zeebe OS Exporter Integration Tests"
+            maven-modules: "'io.camunda:zeebe-opensearch-exporter'"
+            maven-build-threads: 2
+            maven-test-fork-count: 7
+            runs-on: ubuntu-latest
+          - group: camunda-exporter
+            name: "Camunda Exporter Integration Tests"
+            maven-modules: "'io.camunda:camunda-exporter'"
+            maven-build-threads: 2
+            maven-test-fork-count: 7
+            runs-on: ubuntu-latest
+    env:
+      ZEEBE_TEST_DOCKER_IMAGE: localhost:5000/camunda/zeebe:current-test
+    services:
+      registry:
+        image: registry:2
+        ports:
+          - 5000:5000
+    steps:
+      - uses: actions/checkout@v4
+      - name: Import secrets from Vault
+        id: secrets
+        uses: hashicorp/vault-action@d1720f055e0635fd932a1d2a48f87a666a57906c # v3.0.0
+        with:
+          url: ${{ secrets.VAULT_ADDR }}
+          method: approle
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+          secrets: |
+            secret/data/products/camunda/ci/github-actions CAMUNDA_CI_OPENSEARCH_2_9_URL;
+      - uses: ./.github/actions/setup-build
+        with:
+          maven-cache-key-modifier: it-${{ matrix.group }}
+          vault-address: ${{ secrets.VAULT_ADDR }}
+          vault-role-id: ${{ secrets.VAULT_ROLE_ID }}
+          vault-secret-id: ${{ secrets.VAULT_SECRET_ID }}
+      - uses: ./.github/actions/build-zeebe
+        id: build-zeebe
+        with:
+          maven-extra-args: -T1C -PskipFrontendBuild -D skipOptimize
+      - uses: ./.github/actions/build-platform-docker
+        with:
+          repository: localhost:5000/camunda/zeebe
+          version: current-test
+          push: true
+          distball: ${{ steps.build-zeebe.outputs.distball }}
+      - name: Create build output log file
+        run: echo "BUILD_OUTPUT_FILE_PATH=$(mktemp)" >> $GITHUB_ENV
+      - name: Maven Test Build
+        run: >
+          ./mvnw -B -T ${{ matrix.maven-build-threads }} --no-snapshot-updates
+          -D forkCount=${{ matrix.maven-test-fork-count }}
+          -D maven.javadoc.skip=true
+          -D skipUTs -D skipChecks -D skipOptimize
+          -D failsafe.rerunFailingTestsCount=3 -D flaky.test.reportDir=failsafe-reports
+          -Dcamunda.it.opensearch.aws_instance_url=${{ steps.secrets.outputs.CAMUNDA_CI_OPENSEARCH_2_9_URL }}
```

Comment:
Apparently this setting doesn't take effect otherwise the job cannot be successful on Github-hosted runners (which don't have access to the real AWS OpenSearch instance provided under `CAMUNDA_CI_OPENSEARCH_2_9_URL`) - but also thir PR doesn't show if the job is successful, because it is not triggered.

Reply:
The new CI job is failing right now. The error message is `Connect to localhost:443 [localhost/127.0.0.1, localhost/0:0:0:0:0:0:0:1] failed: Connection refused` which to me looks like the test cases assume that the AWS OpenSearch instance is available under `localhost` (which it is not).

I am still wondering if the `camunda.it.opensearch.aws_instance_url` parameter takes any effect. I tried to see if there is any code in the repository that uses/reads this parameter via [GitHub search but there is none](https://github.com/search?q=repo%3Acamunda%2Fcamunda+%22aws_instance_url%22&type=code).

@yevhen-cap where/how did you find that this parameter should be used? Any documentation or another engineer mentioning that it works? I don't see any evidence in the code that this is the right parameter.

Reply:
Merged tests from branch, where this parameter present